easyvista/plugin.spec.yaml

```diff
@@ -0,0 +1,733 @@
+      description: Department label
+      type: string
+      required: false
+  known_error:
+    KNOWNERROR_PATH:
+      title: Known Error Path
+      description: Known error path
+      type: string
+      required: false
+    KNOWN_PROBLEMS_ID:
+      title: Known Problems ID
+      description: Known problems ID
+      type: string
+      required: false
+    KP_NUMBER:
+      title: KP Number
+      description: KP number
+      type: string
+      required: false
+    QUESTION_EN:
+      title: Question EN
+      description: Question EN
+      type: string
+      required: false
+  record:
+    HREF:
+      title: HREF
+      description: HREF hyperlink
+      type: string
+      required: false
+    COMMENT:
+      title: Comment
+      description: Comment
+      type: comment
+      required: false
+    MAX_RESOLUTION_DATE_UT:
+      title: Max Resolution Date
+      description: Max resolution date
+      type: string
+      required: false
+    REQUEST_ID:
+      title: Request ID
+      description: Request ID
+      type: string
+      required: false
+    RFC_NUMBER:
+      title: RFC Number
+      description: RFC number
+      type: string
+      required: false
+    SUBMIT_DATE_UT:
+      title: Submit Date
+      description: Submit date
+      type: string
+      required: false
+    CATALOG_REQUEST:
+      title: Catalog Request
+      description: Catalog request
+      type: catalog_request
+      required: false
+    STATUS:
+      title: Status
+      description: Status
+      type: status
+      required: false
+    RECIPIENT:
+      title: Recipient
+      description: Recipient
+      type: employee
+      required: false
+    REQUESTOR:
+      title: Requestor
+      description: Requestor
+      type: employee
+      required: false
+    LOCATION:
+      title: Location
+      description: Location
+      type: location
+      required: false
+    DEPARTMENT:
+      title: Department
+      description: Department
+      type: department
+      required: false
+    KNOWNERROR:
+      title: Known Error
+      description: Known error
+      type: known_error
+      required: false
+  search_ticket_results:
+    HREF:
+      title: HREF
+      description: HREF hyperlink
+      type: string
+      required: false
+    record_count:
+      title: Record Count
+      description: Record count
+      type: string
+      required: false
+    total_record_count:
+      title: Total Record Count
+      description: Total record count
+      type: string
+      required: false
+    records:
+      title: Records
+      description: Records
+      type: "[]record"
+      required: false
+  ticket_data:
+    href_hyperlink:
+      title: HREF Hyperlink
+      description: URL link (HREF) to the ticket
+      type: string
+      required: false
+    reference_number:
+      title: Reference Number
+      description: Reference number of the ticket
+      type: string
+      required: false
+
+connection:
+  hostname:
```

Comment:
The example looks like a URL rather than a hostname. If so, we should rename this to `url` and update the description and title.